src/main/java/com/agilethought/atsceapi/dto/LoginDataResponse.java

```diff
@@ -0,0 +1,10 @@
+package com.agilethought.atsceapi.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class LoginDataResponse {
```

Comment:
I don't like name for this class. I would prefer "LoginResponse". But it is just a recommendation.